### DISCUSSION TITLE
Mariner for containerized apps in AKS

### DISCUSSION THREAD

**Discussion by adriangonzalezroble** [microsoft/azurelinux#9134] [Q&A]
Hi everyone,

Our team is looking to update our Java and Node.js container images deployed in Azure Kubernetes Services (AKS).

We found this documentation [Java Containers Documentation](https://learn.microsoft.com/en-us/java/openjdk/containers), which recommends using CBL-Mariner for containerized applications. As a result, we are considering using Mariner for our Java applications.

For Node.js, we found this documentation [Node.js Containers Documentation](https://mcr.microsoft.com/en-us/product/cbl-mariner/base/nodejs/about). However, it states that "CBL-Mariner container images are supported only for internal Microsoft use. For more information, please see our [Support Statement](https://github.com/microsoft/azurelinux/blob/2.0/SUPPORT.md)."

In conclusion, we are uncertain about using Mariner for our applications deployed in AKS. Could anyone clarify whether it's advisable to use Mariner in our situation, especially for Node.js?

Thank you!
